### PR TITLE
Update security-features.md

### PR DESCRIPTION
**Pull request by ee61re**
corrected a couple of typos

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/business/security-features.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Advanced features in Microsoft 365 Business are available to help you protect yo
...
 - **Same [Conditional Access](https://docs.microsoft.com/en-us/azure/active-directory/conditional-access/overview) as Azure AD P1 plan**
 
-    Conditional Access can help protect your organization from sign-in risk, access attempts from an unexpected network or locale, access attempts form risky device types, and so on. Conditional Access policies are enforced after the first authentication is completed, and it uses signals from the first authentication event to determine if the attempted access should be approved, denies, or f more proof (such as second form of identification) is required.
+    Conditional Access can help protect your organization from sign-in risk, access attempts from an unexpected network or locale, access attempts form risky device types, and so on. Conditional Access policies are enforced after the first authentication is completed, and it uses signals from the first authentication event to determine if the attempted access should be approved, denies, or if more proof (such as second form of identification) is required.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/security-features.md]:
```suggestion
    Conditional Access can help protect your organization from sign-in risk, access attempts from an unexpected network or locale, access attempts from risky device types, and so on. Conditional Access policies are enforced after the first authentication is completed, and it uses signals from the first authentication event to determine if the attempted access should be approved, denied, or if more proof (such as a second form of identification) is required.
```

File: microsoft-365/business/security-features.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Your Microsoft 365 Business subscription includes features that help you maintai
...
+   Microsoft 365 Business includes all the features of [Azure Information Protection Plan 1](https://go.microsoft.com/fwlink/p/?linkid=871407). With this plan you can create **Sensitivity labels**, that allow you to control access to sensitive information in email and documents with controls like "Do not forward" and "Do not copy". You can also classify sensitive information as "Confidential" and specify how classified information can be shared outside and inside the business. Enterprise-grade encryption is easy to apply to email and documents to keep your information private. You can also install the Azure Information Protection client add-in for Office apps. For more details, see [Azure Information Protection unified labeling client](https://docs.microsoft.com/azure/information-protection/rms-client/unifiedlabelingclient-version-release-history). For Sensitivity labels, install the **AzInfoProtection_UL.exe**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/security-features.md]:
```suggestion
   Microsoft 365 Business includes all the features of [Azure Information Protection Plan 1](https://go.microsoft.com/fwlink/p/?linkid=871407). With this plan you can create **Sensitivity labels** that allow you to control access to sensitive information in email and documents with controls like "Do not forward" and "Do not copy". You can also classify sensitive information as "Confidential" and specify how classified information can be shared outside and inside the business. Enterprise-grade encryption is easy to apply to email and documents to keep your information private. You can also install the Azure Information Protection client add-in for Office apps. For more details, see [Azure Information Protection unified labeling client](https://docs.microsoft.com/azure/information-protection/rms-client/unifiedlabelingclient-version-release-history). For Sensitivity labels, install the **AzInfoProtection_UL.exe**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is just for a typo issue.

Thank you.

cc: @kenwith

**Review by nenonix — APPROVED** [review]

**Pull request merged**